charts/incubator/youtrack/values.yaml

```diff
@@ -0,0 +1,36 @@
+---
+image:
+  repository: jetbrains/youtrack
+  tag: 2022.2.55618
+  pullPolicy: IfNotPresent
+persistence:
+  data:
+    enabled: true
+    mountPath: /opt/youtrack/data
+  conf:
+    enabled: true
+    mountPath: /opt/youtrack/conf
+  logs:
+    enabled: true
+    mountPath: /opt/youtrack/logs
+  backups:
+    enabled: true
+    mountPath: /opt/youtrack/backups
+  temp:
+    enabled: true
+    mountPath: /opt/youtrack/temp
```

Comment:
Any docs for this? I can't find anywhere in their docs that says the temp is in this location

Reply:
Well the install failed as it couldn't write to a temp directory in this location but saw no docs for it. Unless the error was a red herring?

Reply:
* directory /opt/youtrack/temp is not writable